File: Code Chef/Equal Distinct (EQDIS).cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	int t;cin>>t;
	while(t--){
	    int n;cin>>n;
	    int x;
	    set<int>s;
	    for(int i=0;i<n;i++){
	        cin>>x;
	        s.insert(x);
	    }
	    if( s.size()==(n) && n%2!=0){
	        cout<<"NO"<<endl;
	    }
	    else{
	        cout<<"YES"<<endl;
	    }
	}
	return 0;
}

File: Code Chef/Equal Distinct (EQDIS).py
# cook your dish here
t = int(input())
for i in range(t):
    n= int(input())
    l= list(map(int,input().strip().split()) )
    s= set(l)
    if(len(s)== n and n%2!=0):
        print("NO")
    else:
        print("YES")
